src/GraphQL/Types/Schema.cs

```diff
@@ -306,6 +374,9 @@ private void CreateSchemaTypes()
             // However, AllTypes and similar properties will return a reference to SchemaTypes without waiting for a lock.
             _allTypes.ApplyMiddleware(FieldMiddleware);
 
+            foreach (var visitor in GetVisitors())
+                this.Run(visitor);
```

Comment:
These lines unify the way how we handle directives for graph type first and schema first approaches.